python/MooseDocs/extensions/content.py

```diff
@@ -356,12 +356,38 @@ def createHTML(self, parent, token, page):
             # can we fix this?
             for k, v in node['headings'].items():
                 if v['level'] in token['levels'] and k not in token['hide']:
+                    classref = 'moose-outline-{}'.format(v['level'])
+
+                    if v['level'] is 1:
+                        ol1 = html.Tag(ul, 'ol', class_=classref)
+                        li = html.Tag(ol1, 'li')
+                    elif v['level'] is 2:
+                        ol2 = html.Tag(ol1, 'ol', class_=classref)
+                        li = html.Tag(ol2, 'li')
+                    elif v['level'] is 3:
+                        ol3 = html.Tag(ol2, 'ol', class_=classref)
+                        li = html.Tag(ol3, 'li')
+                    elif v['level'] is 4:
+                        ol4 = html.Tag(ol3, 'ol', class_=classref)
+                        li = html.Tag(ol4, 'li')
+                    elif v['level'] is 5:
+                        ol5 = html.Tag(ol4, 'ol', class_=classref)
+                        li = html.Tag(ol5, 'li')
+                    elif v['level'] is 6:
+                        ol6 = html.Tag(ol5, 'ol', class_=classref)
+                        li = html.Tag(ol6, 'li')
+                    # else: error? is this even possible?
+
                     url = pageref + '#{}'.format(k)
                     head = heading.find_heading(node, k)
                     link = core.Link(None, url=url)
                     head.copyToToken(link)
-                    core.LineBreak(link)
-                    self.renderer.render(div, link, page)
+                    self.renderer.render(li, link, page)
```

Comment:
@aeslaughter do you have any suggestions for a cleaner way to do this? We need to open a new `<ol>` tag for each heading level, and they each need a unique class reference so that the counters in the `css` file function properly.